spanner/api/Spanner.Samples.Tests/SpannerFixture.cs

```diff
@@ -514,11 +536,12 @@ public async Task CreateVenuesTableAndInsertDataAsync(string databaseId)
         }));
     }
 
-    public async Task RefillMarketingBudgetsAsync(int firstAlbumBudget, int secondAlbumBudget)
+    public async Task RefillMarketingBudgetsAsync(int firstAlbumBudget, int secondAlbumBudget, string databaseId = null)
     {
+        var spannerConnection = (databaseId == null) ? SpannerConnection : new SpannerConnection($"Data Source=projects/{ProjectId}/instances/{InstanceId}/databases/{databaseId}");
```

Comment:
```suggestion
        var spannerConnection = databaseId is null ? SpannerConnection : new SpannerConnection($"Data Source=projects/{ProjectId}/instances/{InstanceId}/databases/{databaseId}");
```
I don't think we need braces here.